src/components/forms/badge-form.js

```diff
@@ -126,6 +128,27 @@ class BadgeForm extends React.Component {
         this.props.onBadgePrintExport();
     }
 
+    handleBadgePrintClear(ev) {
+        const {entity} = this.state;
+        const {clearBadgePrints} = this.props;
+
+        ev.preventDefault();
+
+        Swal.fire({
+            title: T.translate("general.are_you_sure"),
+            text: `${T.translate("edit_ticket.clear_badge_prints_warning")}`,
+            type: "warning",
+            showCancelButton: true,
+            cancelButtonColor: '#d33',
+            confirmButtonColor: '#3085d6',
+            confirmButtonText: T.translate("general.clear"),
+        }).then(function (result) {
+            if (result.value) {
+                clearBadgePrints(entity.id);
```

Comment:
this entity.id is the badge id u need to pass the ticket id , so here should be , i do think, entity.ticket_id